src/ui/i18n/translator.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * A placeholder for Translation calls handled during preprocessing.
+ *
+ * Any calls of this method will be removed during a preprocessing step during SDK
+ * bundling. To support cases where someone may want to bundle without using our
+ * bundling tasks, this functions simply returns the phrase passed in.
+ *
+ * @param {string} phrase
+ * @param {string} pluralPhrase
+ * @param {string | number} count
+ * @param {string} context
+ * @param {Object} interpolationValues
+ * @returns {string}
+ */
+export function replaceWithTranslation ({ phrase, pluralPhrase, count, context, interpolationValues }) {
```

Comment:
Minor nit: With this, we have 3 files in the SDK called translator.js. Two in src and one in conf. I think it would be good to rename at least one of the usages in src so we can differentiate more easily between those two.

Reply:
Spoke offline and renamed to translationflagger.js! 🎏